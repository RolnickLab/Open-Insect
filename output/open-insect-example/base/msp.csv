TPR@5,FPR@95,AUROC,AUPR,ACC
10.0,30.0,83.0,78.5365,90.0
10.0,30.0,83.0,78.5365,90.0
50.0,30.0,93.0,92.9026,90.0
60.0,30.0,94.0,94.3312,90.0
55.0,30.0,93.5,93.6169,90.0
