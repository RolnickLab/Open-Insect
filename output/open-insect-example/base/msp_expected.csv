TPR@5,FPR@95,AUROC,AUPR,ACC
40.0,90.0,71.0,77.7467,90.0
40.0,90.0,71.0,77.7467,90.0
60.0,30.0,88.0,89.465,90.0
50.0,30.0,87.0,88.0365,90.0
55.0,30.0,87.5,88.7507,90.0
